addons/test_mass_mailing/tests/test_mailing.py

```diff
@@ -378,3 +378,33 @@ def test_mailing_mailing_list_optout(self):
             check_mail=True
         )
         self.assertEqual(mailing.ignored, 2)
+
+    @users('user_marketing')
+    def test_mass_mailing_duplicates(self):
```

Comment:
Checking 14.0 we have a 'test_mailing_w_seenlist_unstored_partner' that exists ... could be 'test_mailing_w_seenlist' simply maybe, and defined before that one ?